ext/imap/php_imap.c

```diff
@@ -526,6 +530,7 @@ PHP_MINIT_FUNCTION(imap)
 	REGISTER_LONG_CONSTANT("OP_SECURE", OP_SECURE, CONST_PERSISTENT | CONST_CS);
 	/* don't do non-secure authentication */
 
+
```

Comment:
This seems to have been added unintentionally

Reply:
Thanks, fixed